test/integration/coronavirus_local_restrictions_test.rb

```diff
@@ -148,7 +148,7 @@ class CoronavirusLocalRestrictionsTest < ActionDispatch::IntegrationTest
       then_i_enter_a_valid_english_postcode_with_a_future_level_four_restriction
       then_i_click_on_find
       then_i_see_the_results_page_for_level_three_with_changing_restriction_levels
-      then_i_do_not_see_details_of_christmas_rules
+      then_i_see_details_of_christmas_rules
```

Comment:
This confused me a bit as I wondered why it had changed. Having a little think I wonder if it's best to remove the Christmas assertions from these restriction changing test since the restriction changing has no impact on that and it's already tested in the various ["displays the tier x restrictions"](https://github.com/alphagov/collections/blob/540fe2c2f0411ab41dccb487bec3fc6a13dd55fd/test/integration/coronavirus_local_restrictions_test.rb#L19) tests.

Reply:
yeah, excellent shot. Have taken them out